archive/s/Scilab/README.md

```diff
@@ -0,0 +1,30 @@
+# What is Scilab
+Scilab is free and open source software for numerical computation providing a powerful computing environment for engineering and scientific applications.
+
```

Comment:
Title this section "Fun Facts" and then put all the sections below (besides references) as a subsection of this section.

```suggestion

## Fun Facts
```